Demo/VIper-v2-Demo/Viper-v2-Demo/Modules/Home/HomeViewController.swift

```diff
@@ -15,16 +15,83 @@ final class HomeViewController: UIViewController {
     // MARK: - Public properties -
 
     var presenter: HomePresenterInterface!
+    
+    @IBOutlet weak var emptyPlaceholderLabel: UILabel!
```

Comment:
It seems that in a lot of view controllers / cell-s you have properties that could be private.